src/components/ha-paper-icon-button-arrow-next.ts

```diff
@@ -0,0 +1,31 @@
+import { Constructor } from "lit-element";
+import "@polymer/paper-icon-button/paper-icon-button";
+// Not duplicate, this is for typing.
+// tslint:disable-next-line
+import { PaperIconButtonElement } from "@polymer/paper-icon-button/paper-icon-button";
+
+const paperIconButtonClass = customElements.get(
+  "paper-icon-button"
+) as Constructor<PaperIconButtonElement>;
+
+export class HaPaperIconButtonArrowNext extends paperIconButtonClass {
+  public connectedCallback() {
+    super.connectedCallback();
```

Comment:
This one still calls super before.

Reply:
Problem was never in these ones! I know it sounds crazy but I think it's related to where it is on the page and this one is always pretty high up as opposed to the non-arrow one. Anyway - I'll make the change to keep it uniform.